lib/evaluateFunctions.js

```diff
@@ -241,19 +242,33 @@ function factorMultiplicands(node) {
   const radicandNode = node.args[0];
 
   let children = [];
+  let factored = false;
   radicandNode.args.forEach(child => {
-    if (NodeType.isConstant(child) && !Negative.isNegative(child)) {
-      const radicandValue = parseFloat(child.value);
-      const factors = Factor.getPrimeFactors(radicandValue);
-      const factorNodes = factors.map(NodeCreator.constant);
-      children = children.concat(factorNodes);
+    if (PolynomialTermNode.isPolynomialTerm(child)) {
+      const polyTerm = new PolynomialTermNode(child);
+      const coeffNode = polyTerm.getCoeffNode();
+      const polyTermNoCoeff = NodeCreator.polynomialTerm(
+        polyTerm.getSymbolNode(), polyTerm.getExponentNode(), null);
+      if (coeffNode) {
+        const factorNodes = getFactorNodes(coeffNode);
+        if (factorNodes.length > 1) {
+          factored = true;
+        }
+        children = children.concat(factorNodes);
+      }
+      children.push(polyTermNoCoeff);
     }
+
```

Comment:
code was feeling crowded - no strong reason
if else should probably be kept together